app/uk/gov/hmrc/individualsmatchingapi/controllers/CommonController.scala

```diff
@@ -73,18 +73,17 @@ trait PrivilegedAuthentication extends AuthorisedFunctions {
   def authPredicate(scopes: Iterable[String]): Predicate =
     scopes.map(Enrolment(_): Predicate).reduce(_ and _)
 
-  def requiresPrivilegedAuthentication(endpointScopes: Iterable[String])(
-    implicit hc: HeaderCarrier): Future[List[String]] = {
+  def requiresPrivilegedAuthentication(endpointScopes: Iterable[String])(f: Iterable[String] => Future[Result])(
```

Comment:
Should the common controller be in a V2 namespace to ensure it is never pulled into V1?